### PR TITLE
React to inprocess mode removal

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs [hunk 1]
```diff
@@ -57,10 +57,8 @@ public Task ResponseCompression_IISExpress_NoCompression(RuntimeFlavor runtimeFl
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
         [InlineData(RuntimeFlavor.CoreClr, ApplicationType.Portable, HostingModel.OutOfProcess, "/p:ANCMVersion=V1")]
         [InlineData(RuntimeFlavor.CoreClr, ApplicationType.Portable, HostingModel.OutOfProcess, "/p:ANCMVersion=V2")]
```
Comment (Tratcher) [inline test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: Shouldn't you also pull all of the v2 tests?

Comment (pakrym) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: Why?

Comment (jkotalik) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: V2 isn't being shipped, so no need for the tests?

Comment (pakrym) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: It's still being built, and tests work, why remove?

Comment (jkotalik) [reply test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs]: I'm fine with keeping it in.

**Comment by pakrym** [comment]
They don't seem to fail on our CI so I'll try merging and see how it would go.

**Pull request merged**